clang/bindings/python/tests/cindex/test_lib.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import clang.cindex
+
+if "CLANG_LIBRARY_PATH" in os.environ:
+    clang.cindex.Config.set_library_path(os.environ["CLANG_LIBRARY_PATH"])
+
+import unittest
+import ast
+
+
+class TestLib(unittest.TestCase):
+    def test_functions_registered(self):
+        def get_function_spelling(node):
+            # The call expressions we are interested in have their spelling in .attr, not .id
```

Comment:
It seems that darker is not run on tests, but we still shouldn't exceed 88 (?) columns without a good justification